src/util/Scaling.h

```diff
@@ -0,0 +1,11 @@
+//
+// Created by rolf on 16-11-19.
+//
+
+#ifndef RTT_SCALING_H
+#define RTT_SCALING_H
+
+inline static double mmToM(double unit){
```

Comment:
This exists in utils already

Reply:
It does. It's however only defined for floats (not for doubles sadly), so we either need to make it a template or also add a double version there.